### ISSUE TITLE
Dual screen

### ISSUE THREAD

**Issue by renaudbrion** [saenzramiro/rambox#323]
Hi, first at all, thank you for this wonderful app ! Keep going 👍 

I use 2 screens with my laptop closed. The Rambox window is on the secondary screen. When I use the laptop without external screen I can't display Rambox even if it is properly running. 

Thank you in advance.
Kinds Regards

## 

> Rambox 0.4.2
> Electron 1.3.5
> win32 x64 10.0.10240

**Comment by MrChick** [comment]
As a workaround until that bug is fixed:
Press Win+Right to move the window to the proper screen

**Comment by renaudbrion** [comment]
Thx, it works :-)

**Comment by saenzramiro** [comment]
If I'm not wrong, is behavior is for all softwares. Right?

**Comment by renaudbrion** [comment]
Indeed, the "fix" works for all softwares. However, I have this issue only with Rambox. Even if I reboot my laptop the software stays on the "ghost" screen... 👻

**Comment by MrChick** [comment]
@saenzramiro That problem basically exists for every software, yes. But most software seems to check if the position/the screen they're trying to get shown on actually exists when they're starting, and if not, move to some default

**Comment by saenzramiro** [comment]
This is weird, I work with 3 monitors and for me works ok. Seems like @MrChick said, Windows handle the positioning of the window.

My environment:    |    1    |    2*    |    3    |
* main monitor

I have Rambox in 1, so if I open Rambox to see it in 1 and then disable monitor 1, it will moves to 2. Or if I have Rambox closed in systray, then disable 1 and open Rambox, it will shows in 2.

**Comment by raulmatei** [comment]
Hi,

I've just had a similar issue on MacOS. It was working fine, it was moved out of the second screen when I disconnected the external display. I continued working on my MacBook display for a day, without the external display attached and Rambox worked just fine. Then last night I've had to restart the system, to apply some updates and Rambox started in a "ghost" mode, the App menu was fine, I was able to see messages in the dock icon and top right notification worked, but the main window was hidden. 

I've tried all I knew and all I could found on the web, nothing worked... then I clicked on the Rambox menu in app bar and clicked on About Rambox item, suddenly the window appeard and all is fine now... Hope it helps other OSX users.

Raul.
